<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditChannelPropertiesDialog</class>
 <widget class="QDialog" name="EditChannelPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="displayNameLabel">
         <property name="toolTip">
          <string>The name that is displayed in the UI for the channel</string>
         </property>
         <property name="text">
          <string>Channel Display Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="displayNameValue"/>
       </item>
       <item>
        <widget class="QLabel" name="channelHeaderLabel">
         <property name="toolTip">
          <string>Markdown-formatted text to display in the header of the channel</string>
         </property>
         <property name="text">
          <string>Channel Header:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="channelHeaderValue">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="purposeLabel">
         <property name="toolTip">
          <string>A short description of the purpose of the channel</string>
         </property>
         <property name="text">
          <string>Channel Purpose:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="purposeValue">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditChannelPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditChannelPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
